checks.d/mysql.py

```diff
@@ -97,8 +97,8 @@ def _get_config(self, instance):
 
     def _connect(self, host, port, mysql_sock, user, password, defaults_file):
         service_check_tags = [
-            'host:%s' % host,
-            'port:%s' % port
+            'host:%s' % mysql_sock if mysql_sock != '' else host,
```

Comment:
Wondering if this should be `localhost` if `mysql_sock` is set, given that a sock connection would always be local... not sure.